.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @dsvoronin @eugene-krivobokov @sboishtyan
+* @dsvoronin @eugene-krivobokov @sboishtyan @RuslanMingaliev
```

Comment:
And to maintainers list in `convention.publish-base.gradle.kts`, please

Reply:
done